advanced/am-pattern-1/templates/am/wso2am-pattern-1-am-volume-claim.yaml

```diff
@@ -23,14 +23,7 @@ spec:
   resources:
```

Comment:
@Chathuru Let us rename this files to `wso2am-pattern-1-am-volume-claims.yaml` as it contains multiple PVCs.

Reply:
Renamed the file in https://github.com/wso2/kubernetes-apim/pull/365/commits/656646538e8cc1b8d8fbc51cb3d206a1c58101e5